registration/views.py

```diff
@@ -549,6 +567,21 @@ def get_context_data(self, form, **kwargs):
         d['postal_code'] = self.request.session[SESSION_POSTAL_CODE]
         d['email'] = self.request.session[SESSION_EMAIL]
 
+        current_form = self.form_list[unicode(self.steps.current)]
+        if 'license_id' in current_form.base_fields:
+            if SESSION_LICENSE_ID_FORMATS in self.request.session:
+                license_id_formats = strip_unicode_from_list(
+                    self.request.session[SESSION_LICENSE_ID_FORMATS])
+                d['license_id_formats'] = json.dumps(license_id_formats)
+                invalid_license_model_data = {
+                    'title': _('Please check your license ID'),
+                    'body': _('The license ID you entered doesn&rsquo;t seem to match the format of license IDs '
+                              'for this State. Please double-check it, and if you&rsquo;re sure its right, '
+                              'please choose to continue.'),
+                    'ok': _('Continue to Next Page'),
+                    'cancel': _('Re-check License ID'), }
+                d['invalid_license_model_data'] = invalid_license_model_data
```

Comment:
@mabuaisha not important to change, but this should be ``modal`` with an "a" not ``model``